src/main/java/org/maxgamer/quickshop/nonquickshopstuff/com/sk89q/worldedit/util/paste/EngineHubPaster.java

```diff
@@ -1,78 +0,0 @@
-/*
```

Comment:
Should keep this file in project so we can reuse those sources code in future if we decide host same paste program by our self.

Reply:
just comment whole file